readme.md

```diff
@@ -94,17 +97,141 @@ Visit <https://indiafightscorona.giveindia.org> and make a small donation to hel
     -   [Wakatime Card Exclusive Option](#wakatime-card-exclusive-options)
 -   [Deploy Yourself](#deploy-on-your-own-vercel-instance)
     -   [Keep your fork up to date](#keep-your-fork-up-to-date)
+    
+# Getting Started
+
+## GitHub Actions (Recommended)
+
+Using GitHub Actions over the community deployment is recommended because it provides more accurate stats and 100% uptime. Simply copy-paste the following into the `/.github/workflows/grs.yml` file in your README repo (`USERNAME/USERNAME`).
+
+```yml
+name: GitHub Readme Stats
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: 0 * * * *
```

Comment:
@anuraghazra, @Zo-Bro-23 do you think we should suggest users run it every hour? Let's do a quick check:

- It takes about 30 seconds to generate two cards; let's make it 1 min to be sure.
- The free GitHub plan includes 2000 CI minutes per month.
- You will use 24*30.437=730.488 minutes for GRS card generation at this rate.

Let's try running it every 4 hours, like the cache time of the current Vercel version. This will account for 182.622 CI minutes. Every 2 hours should also be okay.

```suggestion
  - cron: 0 */4 * * *
```

Reply:
> GitHub Actions usage is free for standard GitHub-hosted runners in public repositories, and for self-hosted runners.

I use a lot of workflows, but my profile always shows 0 minutes used. This is likely why. Since profile READMEs are anyways public, this will not be an issue. Moreover, private repos have 2000 minutes included for free, so 730 minutes is fine. Users can change this if they want. Should I include that as a note in the README that they can change it?

Reply:
@ Zo-Bro-23 I have maxed out my free budget more than once with the number of actions running in my repositories. I, therefore, have a pro subscription. Also, keep into account that if we enable multi-page fetching and the organization function, the run time will increase significantly for some relentless GitHub users. Let's see what Anurag thinks.

Reply:
@anuraghazra, @Zo-Bro-23 just pointed out to me that the GitHub documentation states that the Github action minutes are not charged for public repos that use standard single core actions (see https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions). Resolving this.